tiledb/sm/c_api/tiledb.cc

```diff
@@ -479,171 +480,11 @@ int32_t tiledb_domain_get_ndim(
 
 int32_t tiledb_domain_add_dimension(
     tiledb_ctx_t* ctx, tiledb_domain_t* domain, tiledb_dimension_t* dim) {
-  if (sanity_check(ctx) == TILEDB_ERR ||
-      sanity_check(ctx, domain) == TILEDB_ERR)
-    return TILEDB_ERR;
-
-  throw_if_not_ok(domain->domain_->add_dimension(
-      make_shared<tiledb::sm::Dimension>(HERE(), dim->dim_)));
-
-  return TILEDB_OK;
-}
-
-int32_t tiledb_domain_dump(
-    tiledb_ctx_t* ctx, const tiledb_domain_t* domain, FILE* out) {
-  if (sanity_check(ctx) == TILEDB_ERR ||
-      sanity_check(ctx, domain) == TILEDB_ERR)
-    return TILEDB_ERR;
-  domain->domain_->dump(out);
-  return TILEDB_OK;
-}
-
-/* ********************************* */
-/*             DIMENSION             */
-/* ********************************* */
-
-int32_t tiledb_dimension_alloc(
-    tiledb_ctx_t* ctx,
-    const char* name,
-    tiledb_datatype_t type,
-    const void* dim_domain,
-    const void* tile_extent,
-    tiledb_dimension_t** dim) {
-  if (sanity_check(ctx) == TILEDB_ERR)
+  if (sanity_check(ctx, domain) == TILEDB_ERR)
```

Comment:
Minor nit: Shouldn't we add braces around the body since we're modifying the condition?

Reply:
Done.